canonical-web-and-design/ubuntu.com
# [Certified improvements] - add functionality for expandable section

## Done

- Add option for accordion to have multiple tabs open if the `data-multiple-expanded` attribute is set to true
- NOTE: There will be functionality added in a future PR to only see 5 filters per group, and click "See all" to expand each group. As this is not in the scope of this PR, I have left the "Vendors" group closed on page load as the list of vendors is very long and users may miss the "update" button. In the PR where the "See all" functionality is added I will ensure all 3 groups are expanded on page load. 

## QA

- View page at: https://ubuntu-com-10066.demos.haus/certified?q=dell
- "Category" and "Certified Ubuntu release" groups should be expanded on page load, and "Vendors" should be collapsed. 
-  Click on the titles of each group and check they open and close as expected


## Issue / Card

Fixes [#4248](https://github.com/canonical-web-and-design/web-squad/issues/4248)

## Screenshots

<img width="1218" alt="Screenshot 2021-07-22 at 16 40 01" src="https://user-images.githubusercontent.com/58959073/126667632-9c2a6df3-3a57-403f-b3c0-240c20597ce0.png">
